test/config.test.js

```diff
@@ -270,6 +270,37 @@ describe('javascript config', () => {
 
 });
 
+
+describe('`logger` config', () => {
+    it('should exist', () => {
```

Comment:
There are no tests to check that the values can be updated?

Reply:
Added.